x/evm/types/msg.go

```diff
@@ -454,3 +455,28 @@ func deriveChainID(v *big.Int) *big.Int {
 	v = new(big.Int).Sub(v, big.NewInt(35))
 	return v.Div(v, big.NewInt(2))
 }
+
+// Return tx sender and gas price
+func (msg MsgEthereumTx) GetTxInfo(ctx sdk.Context) mempool.ExTxInfo {
+	exTxInfo := mempool.ExTxInfo{
+		Sender:   "",
+		GasPrice: big.NewInt(0),
+		Nonce:    msg.Data.AccountNonce,
+	}
+
+	chainIDEpoch, err := types.ParseChainID(ctx.ChainID())
+	if err != nil {
+		return exTxInfo
```

Comment:
The caller could process correctly that even the members of exTxInfo.Sender and  exTxInfo.GasPrice was empty ?

Reply:
this will be deal in tendermint layer